features/evidence_hub_search.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Evidence Hub Search
+  As a website user
+  I want to be able to see a brief summary of all the insight summaries
```

Comment:
Is this the format being used now as opposed to 
```
As a website user
In order to see the evidence in the hub
I want to be able to see a brief summary of all the insight summaries
```

Reply:
Since it is the first one, I didn't know about the format being used. Should we base this on the one that we did on wpcc? (I am ok to do so, just asking).